src/lib/styles/brand-themes.css

```
@@ -0,0 +1,129 @@
+ *
+ * Delete this file + its import in +layout.svelte to remove the experiment.
+ * ============================================================================ */
+
+@import '@fontsource-variable/nata-sans';
+
+/* Shared across both candidate themes. Nata Sans applies ONLY here, so the
+ * typeface is part of the A/B comparison against Legacy's system stack. */
+[data-brand='a'],
+[data-brand='b'] {
+	--font-brand: 'Nata Sans Variable', ui-sans-serif, system-ui, -apple-system, sans-serif;
+	font-family: var(--font-brand);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)